cosmos/operators/local.py

```diff
@@ -1009,5 +1012,5 @@ def __init__(self, **kwargs: str) -> None:
 
 class DbtCompileLocalOperator(DbtCompileMixin, DbtLocalBaseOperator):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
-        kwargs["should_upload_compiled_sql"] = True
+        kwargs["should_upload_compiled_sql"] = False
```

Comment:
```suggestion
        kwargs["should_upload_compiled_sql"] = True
```
to update before merging